tests/utils/Parsers.ts

```diff
@@ -0,0 +1,4 @@
+export const assetToNumber = (asset: string): number =>  {
```

Comment:
We already have something like this:
https://github.com/hypha-dao/dao-contracts/blob/develop/tests/types/Asset.ts#L13
and you can get the number from the `amount` attribute.

Reply:
Thanks, I didn't notice it before